addons/search-my-stuff/userscript.js

```diff
@@ -22,12 +22,32 @@ export default async function ({ addon, global, console, msg }) {
 
   // Keyboard shortcut for focusing search bar
   document.addEventListener("keypress", (e) => {
+    // Ignore input
+    // 1) in <input> or <textarea>
+    // 2) with Ctrl/Alt/Meta keys pressed (probably browser shortcut keys)
+    // 3) that is a space character (browser shortcut key for paging down)
+    // 4) that is not represented by one alphanumeric key code (probably special letters)
```

Comment:
Note that we're using keypress instead of keydown, so browsers already ignore some of these.